configs/config.json.cluster.example

```diff
@@ -22,23 +22,6 @@
         "type": "HandleSoftBan"
       },
       {
-        "type": "CompleteTutorial",
```

Comment:
I see we have it in another place now. That's fine with me. 

Side note, we should probably make sure the nickname is accepted.